command/plugins_remove.go

```diff
@@ -95,6 +96,11 @@ func (c *PluginsRemoveCommand) RunContext(buildCtx context.Context, args []strin
 			c.Ui.Error(err.Error())
 			return 1
 		}
+		shasumFile := fmt.Sprintf("%s_SHA256SUM", installation.BinaryPath)
+		if err := os.Remove(shasumFile); err != nil {
+			c.Ui.Error(fmt.Sprintf("failed to remove %s: %s", shasumFile, err))
+			c.Ui.Error("You may need to remove it manually")
```

Comment:
This is a nice touch. My only nit is that it gets printed multiple times if we have multiple failed removal but this is probably not an issue.

Reply:
Can we remove multiple plugins at one with the command?

Reply:
If I recall, when you have multiple versions of the same plugin it removes all installed versions. Or am I mistaken?

Reply:
That's true yep, I forgot about this use case. So yeah indeed, it will print the message as much as it fails to remove, but I would argue this is good? If you have a subset of files that fail to be removed for whatever reason, you should know which ones failed to be removed.

Reply:
Yeah - it was a nit not a rejection 😉